### ISSUE TITLE
[Accessibility]: Batches preview: Search bar has no label

### ISSUE THREAD

**Issue by eseliger** [sourcegraph/sourcegraph#34542]
### Audit type

ARC Toolkit

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34080

### Problem description

![CleanShot 2022-04-26 at 22 52 12@2x](https://user-images.githubusercontent.com/19534377/165390364-9d8f1790-87ee-4315-88af-dd472ea59ce2.png)

### Expected behavior

It passes the ARC check.

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
